### ISSUE TITLE
I2PBoteMain, MigrateTo028: Unparseable email identity (Trac #1693)

### ISSUE THREAD

**Issue by str4d** [i2p/i2p.i2p-bote#69]
27/10/15 **:**:** CRIT  [iTest client] t.i2p.router.web.PluginStarter: Error starting up the client class net.i2p.jircii.GuiTest
java.lang.reflect.InvocationTargetException
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at net.i2p.router.startup.LoadClientAppsJob$RunApp.run(LoadClientAppsJob.java:291)
	at java.lang.Thread.run(Thread.java:745)
	at net.i2p.util.I2PThread.run(I2PThread.java:115)
Caused by: java.lang.RuntimeException: Cannot run the graphical application. I2P must be running on a graphical display, and it must be running as the logged-in user and not as a daemon.
	at net.i2p.jircii.GuiTest.main(GuiTest.java:32)
	... 7 more
27/10/15 09:14:55 ERROR [I2PBoteMain ] 2p.bote.migration.MigrateTo028: Unparseable email identity: <#Tue Oct 27 08:12:31 GMT 2015>
27/10/15 09:14:55 ERROR [I2PBoteMain ] 2p.bote.migration.MigrateTo028: Unparseable email identity: <identity0.configuration.includeInGlobalCheck=true>
27/10/15 09:14:55 ERROR [I2PBoteMain ] 2p.bote.migration.MigrateTo028: Unparseable email identity: <identity0.description=******>
27/10/15 09:14:55 ERROR [I2PBoteMain ] 2p.bote.migration.MigrateTo028: Unparseable email identity: <identity0.key=**********>
27/10/15 09:14:55 ERROR [I2PBoteMain ] 2p.bote.migration.MigrateTo028: Unparseable email identity: <identity0.picture=>
27/10/15 09:14:55 ^^^ 2 similar messages omitted ^^^
27/10/15 09:14:55 ERROR [I2PBoteMain ] 2p.bote.migration.MigrateTo028: Unparseable email identity: <identity0.salt=>
27/10/15 09:14:55 ERROR [I2PBoteMain ] 2p.bote.migration.MigrateTo028: Unparseable email identity: <identity0.text=>

Migrated from https://trac.i2p2.de/ticket/1693
```json
{
    "status": "closed", 
    "changetime": "2015-11-01T21:29:45", 
    "description": "27/10/15 **:**:** CRIT  [iTest client] t.i2p.router.web.PluginStarter: Error starting up the client class net.i2p.jircii.GuiTest\njava.lang.reflect.InvocationTargetException\n\tat sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\tat sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)\n\tat sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat java.lang.reflect.Method.invoke(Method.java:606)\n\tat net.i2p.router.startup.LoadClientAppsJob$RunApp.run(LoadClientAppsJob.java:291)\n\tat java.lang.Thread.run(Thread.java:745)\n\tat net.i2p.util.I2PThread.run(I2PThread.java:115)\nCaused by: java.lang.RuntimeException: Cannot run the graphical application. I2P must be running on a graphical display, and it must be running as the logged-in user and not as a daemon.\n\tat net.i2p.jircii.GuiTest.main(GuiTest.java:32)\n\t... 7 more\n27/10/15 09:14:55 ERROR [I2PBoteMain ] 2p.bote.migration.MigrateTo028: Unparseable email identity: <#Tue Oct 27 08:12:31 GMT 2015>\n27/10/15 09:14:55 ERROR [I2PBoteMain ] 2p.bote.migration.MigrateTo028: Unparseable email identity: <identity0.configuration.includeInGlobalCheck=true>\n27/10/15 09:14:55 ERROR [I2PBoteMain ] 2p.bote.migration.MigrateTo028: Unparseable email identity: <identity0.description=******>\n27/10/15 09:14:55 ERROR [I2PBoteMain ] 2p.bote.migration.MigrateTo028: Unparseable email identity: <identity0.key=**********>\n27/10/15 09:14:55 ERROR [I2PBoteMain ] 2p.bote.migration.MigrateTo028: Unparseable email identity: <identity0.picture=>\n27/10/15 09:14:55 ^^^ 2 similar messages omitted ^^^\n27/10/15 09:14:55 ERROR [I2PBoteMain ] 2p.bote.migration.MigrateTo028: Unparseable email identity: <identity0.salt=>\n27/10/15 09:14:55 ERROR [I2PBoteMain ] 2p.bote.migration.MigrateTo028: Unparseable email identity: <identity0.text=>", 
    "reporter": "djullius", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1446413385720174", 
    "component": "apps/plugins", 
    "summary": "I2PBoteMain, MigrateTo028: Unparseable email identity", 
    "priority": "major", 
    "keywords": "I2P-Bote", 
    "version": "0.9.22", 
    "parents": "", 
    "time": "2015-10-27T08:31:42", 
    "milestone": "0.9.23", 
    "owner": "str4d", 
    "type": "defect"
}
```

**Comment by str4d** [comment]
Trac update at 20151027T08:43:06: djullius commented:

> Add a subticket https://trac.i2p2.de/ticket/1695.

**Comment by str4d** [comment]
Trac update at 20151030T02:18:19: 
* zzz commented:

> See https://trac.i2p2.de/ticket/1695 for more info about the unrelated jircii error.
> 
> Not clear what https://trac.i2p2.de/ticket/1695 is for other than a later OOM.
> 


* zzz changed owner from "" to "str4d"
* zzz changed status from "new" to "assigned"

**Comment by str4d** [comment]
Trac update at 20151101T10:24:39: str4d commented:

> Remove a subticket https://trac.i2p2.de/ticket/1695.

**Comment by str4d** [comment]
Trac update at 20151101T10:28:30: 
* str4d commented:

> Fixed in `b58d126b0caeaf4de7b415a4d48b223484588946`.


* str4d changed keywords from "I2PBoteMain, MigrateTo028, Unparseable, email, identity" to "I2P-Bote"
* str4d changed resolution from "" to "fixed"
* str4d changed status from "assigned" to "closed"

**Comment by str4d** [comment]
Trac update at 20151101T21:29:45: str4d changed milestone from "0.9.25" to "0.9.23"
